preprocessing/structs.ts

```diff
@@ -1,15 +1,82 @@
 /* # Mangrove Summary
-   * The Mangrove holds order books for `outbound_tkn`,`inbound_tkn` pairs.
-   * Offers are sorted in a doubly linked list.
+   * Mangrove holds offer lists for `outbound_tkn`,`inbound_tkn` pairs with a given `tickSpacing`.
+   * Offers are sorted in a tree (the "tick tree") where each available price point (a `bin`) holds a doubly linked list of offers.
    * Each offer promises `outbound_tkn` and requests `inbound_tkn`.
    * Each offer has an attached `maker` address.
-   * In the normal operation mode (called Mangrove for Maker Mangrove), when an offer is executed, we:
+   * When an offer is executed, Mangrove does the following:
      1. Flashloan some `inbound_tkn` to the offer's `maker`.
      2. Call an arbitrary `execute` function on that address.
      3. Transfer back some `outbound_tkn`.
      4. Call back the `maker` so they can update their offers.
     
    **Let the devs know about any error, typo etc, by contacting 	devs@mangrove.exchange**
```

Comment:
should we still write an explicit email here?

Reply:
why not?

Reply:
Since we have a whole page (with docs), discord, twitter, etc. it seems like a weird place to send comments. But I don't mind :)

Reply:
We can also add a link but a direct contact point is always pleasant :) Link to https://docs.mangrove.exchange/ ?

Reply:
Yes, I think that is a good idea.

Reply:
done in 74a0f6eb91856a812fcd67c6e856ce0b64ba66ea